scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -307,7 +307,7 @@ class FormatOps(val tree: Tree, val initStyle: ScalafmtConfig) {
         curr = next(curr)
       }
     }
-    result.result()
+    result.result().toSet
```

Comment:
We can keep the mutable set, we won't mutate outside the function anyways. I think you can upcast it to `scala.collection.Set[T]` to remove the mutable operations.